Automattic/vip-go-mu-plugins
# Update Parse.ly to 2.6.1

## Description

This PR pulls the latest version of Parse.ly on the 2.6 branch, 2.6.1.

This is a patch release that contains a fix that prevented the Recommended Widget to work as it was intended.

## Changelog Description

### Plugin Updated: Parse.ly 2.6.1

We upgraded Parse.ly 2.6 to 2.6.1.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR and enable Parse.ly.
2. Create a Recommended Widget without showing the author's name.
3. Check that the author's name doesn't appear.
